src/Manager/Environment/EnvironmentMenu.jsx

```diff
@@ -57,6 +58,23 @@ const openCmd = directory => {
     exec(`start cmd /k "${path.resolve(directory, 'git-cmd.cmd')}"`);
 };
 
+const openTerminal = directory => {
+    const d = path.dirname(directory);
+    const gitversion = readdirSync(`${d}/toolchain/Cellar/git`).pop();
+    const env = [
+        `export PATH=${d}/toolchain/bin:$PATH`,
+        `export GIT_EXEC_PATH=${d}/toolchain/Cellar/git/${gitversion}/libexec/git-core`,
+        'export ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb',
+        `export GNUARMEMB_TOOLCHAIN_PATH=${d}/toolchain`,
+    ];
+    exec(`osascript <<END
+tell application "Terminal"
+    do script "cd ${d} ; ${env.join(' ; ')} ; clear"
+    activate
+end tell
+END`);
```

Comment:
Need indentation above?

Reply:
No, this is how strings templates work in JS.